### ISSUE TITLE
`min`/`max` do not work in some cases

### ISSUE THREAD

**Issue by inumanag** [seq-lang/seq#126]
This:

``` 
def foo(i, m=10, mm=-10, g=-20):
     e = min(i + m, i + mm, i + g)
     print e
foo(10)
```

errors with `could not deduce type parameters of class 'min'`.

**Comment by arshajii** [comment]
https://github.com/seq-lang/seq/commit/7720b36d01e2d3e524c201b10d15b3936a9cc3f7 adds support for up to 5 arguments. Beyond this we'll need vararg support (i.e. `*arg`).

**Comment by inumanag** [comment]
Fully resolved in #132
